shared/protocol/src/main/java/io/pravega/shared/protocol/netty/WireCommandType.java

```diff
@@ -117,6 +117,8 @@
     READ_TABLE_ENTRIES_DELTA(88, WireCommands.ReadTableEntriesDelta::readFrom),
 
     CONDITIONAL_BLOCK_END(89, WireCommands.ConditionalBlockEnd::readFrom),
+    MERGE_SEGMENTS_BATCH(90, WireCommands.MergeSegmentsBatch::readFrom),
+    SEGMENTS_MERGED_BATCH(91, WireCommands.SegmentsMergedBatch::readFrom),
```

Comment:
The convention is VerbNoun -> NounVerbed. So it should be "SegmentsBatchMerged".

Reply:
Ok. Will change this.